tt_metal/impl/buffers/buffer.hpp

```diff
@@ -289,7 +289,7 @@ BufferPageMapping generate_buffer_page_mapping(const Buffer &buffer);
 
 namespace detail {
 using PageAddress = uint32_t;
-using Deviceid = uint32_t;
+using Deviceid = umd::chip_id;
```

Comment:
I can't find how chip_id is defined
I see forward declarations and this
https://github.com/tenstorrent/tt-umd/compare/9d60964a8fe304dca5f6444d3b70b3b5c1a75967...360f6d424d4d9e73d4b266cd163b5b654913b72e#diff-b823f661b8191a05edb5ac40ae81ef2cc3f89ac3017a6408711efa73e72d1c55R17

Can you help to find the full definition?